wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/commands/parallelcommandgroup/ReplaceMeParallelCommandGroup.java

```diff
@@ -0,0 +1,23 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package edu.wpi.first.wpilibj.commands.parallelcommandgroup;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+
+// NOTE:  Consider using this command inline, rather than writing a subclass.  For more
+// information, see:
+// https://docs.wpilib.org/en/latest/docs/software/commandbased/convenience-features.html
+public class ReplaceMeParallelCommandGroup extends ParallelCommandGroup {
+  /**
+   * Creates a new ReplaceMeParallelCommandGroup.
+   */
+  public ReplaceMeParallelCommandGroup() {
+    //Add your commands in the super() call
```

Comment:
It would be helpful to have an example, eg super(new ExampleCommand1(), new ExampleCommand2()); I didn't see anything in frc-docs